.github/workflows/abicheck.ignore.yaml

```diff
@@ -3,17 +3,15 @@ name: ABI
 on:
   pull_request:
     types: [opened, synchronize, reopened]
-    paths:
-      - "**.md"
-      - ".github/ISSUE_TEMPLATE/**"
-      - ".github/workflows/**"
-      - "!.github/workflows/abicheck.yaml"
-      - .clang-format
-      - .gitattributes
-      - .github/dependabot.yaml
-      - .gitignore
-      - .shellcheckrc
-      - LICENSE
+    paths-ignore:
```

Comment:
If anyone else can think of files that might affect ABI, please let me know!

Reply:
What exactly does abi check do?

Reply:
Checks to make sure ABI wasn't broken in stable branches.